defaults/main.yml

```diff
@@ -55,6 +55,37 @@ icinga2_master_usergroups:
 #      # - group1
 #      # - group2
 
+# Icinga2 Hostgroups
```

Comment:
I think the better approach is to define an empty list `[]` as the default value. The current values can then be defined as an example in the comment above the variable.

Reply:
ack